JavaAssignments/src/shraddha/LargestofThreeNos.java

```diff
@@ -0,0 +1,42 @@
+//-------------------------------------------------------------------------------
+// Assignment: Find the largest of three numbers using Logical And operator.
+//-------------------------------------------------------------------------------
+package shraddha;
+
+import java.util.Scanner;
+
+public class LargestofThreeNos {
+	
+	void largestofThreeNos(int first_number, int second_number,int third_number )
+	{
+		if (first_number > second_number && first_number > third_number)
+		{
+			System.out.println("First Number is greatest");
+		} 
+		else if (second_number > first_number && second_number > third_number)
```

Comment:
Here you dont need to compare second_number with first_number again. Think about it.